api/models.py

```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```

Comment:
Sort by SC, LC, AC, MC is much more difficult, than 01, 02, 03, 04.. Why we must use these codes?